demo/vagrant/jobs/loki.nomad

```diff
@@ -0,0 +1,101 @@
+job "loki" {
+  datacenters = ["dc1"]
+
+  group "loki" {
+    count = 1
+
+    task "loki" {
+      driver = "docker"
+
+      config {
+        image = "grafana/loki:1.5.0"
+
+        args = [
+          "--config.file=/etc/loki/config/loki.yml",
+        ]
+
+        network_mode = "host"
+
+        volumes = [
+          "local/config:/etc/loki/config",
+        ]
+
+        port_map {
+          loki_port = 3100
+        }
+      }
+
+      template {
+        data = <<EOH
+---
+auth_enabled: false
+
+server:
+  http_listen_port: 3100
+
+ingester:
+  lifecycler:
+    address: 127.0.0.1
+    ring:
+      kvstore:
+        store: inmemory
+      replication_factor: 1
+    final_sleep: 0s
+  chunk_idle_period: 5m
+  chunk_retain_period: 30s
+
+schema_config:
+  configs:
+  - from: 2020-05-15
+    store: boltdb
+    object_store: filesystem
+    schema: v11
+    index:
+      prefix: index_
+      period: 168h
+
+storage_config:
+  boltdb:
+    directory: /tmp/loki/index
+
+  filesystem:
+    directory: /tmp/loki/chunks
+
+limits_config:
+  enforce_metric_name: false
+  reject_old_samples: true
+  reject_old_samples_max_age: 168h
+EOH
+
+        change_mode   = "signal"
+        change_signal = "SIGHUP"
+        destination   = "local/config/loki.yml"
+      }
+
+      resources {
+        cpu    = 100
+        memory = 256
+
+        network {
+          mbits = 10
+
+          port "loki_port" {
+            static = 3100
```

Comment:
I am not too familiar with Loki. Does this port provide any useful information for users, like an API or CLI? 

If it does, it would be nice to have it exposed in the `Vagrantfile`, otherwise we could use a dynamic port rather than `static`.

Reply:
the host network and static port was an artifact of previous work where i posed myself the same question.

Both loki and promtail do expose a http endpoint [1][2] that exposes an api and health endpoints for loki and a simple dashboard (shows what files it will scrape) and 2 health points for promtail. 

IMHO exposing those would have distracted from the story the demo wants to tell, so i opted not the expose but forgot to cleanup after myself. If you feel you want to expose those (and then promtail makes most sense) i am happy to do the work and expose them.

[1] https://github.com/grafana/loki/blob/master/docs/api.md
[2]https://github.com/grafana/loki/blob/master/docs/clients/promtail/README.md#api

Reply:
Thanks for the extra info. I agree that we don't need these endpoints for the demo 👍